libr/anal/class.c

```diff
@@ -815,10 +815,13 @@ R_API RAnalClassErr r_anal_class_vtable_get(RAnal *anal, const char *class_name,
 
 	vtable->offset = r_num_math (NULL, cur);
 
-	cur = next;
-	sdb_anext (cur, NULL);
-
-	vtable->size = r_num_math (NULL, cur);
+	if (next != NULL) {
+		cur = next;
+		sdb_anext (cur, NULL);
+		vtable->size = r_num_math (NULL, cur);
```

Comment:
i think r_num_get is enough, no need to compute math expressions here.

Reply:
hmm i really don't know this stuff i just enclosed it in an IF, but i can try to change it and see that nothing breaks as long as it's in r2r